### DISCUSSION TITLE
Default PDF ticket template attached to email template

### DISCUSSION THREAD

**Discussion by JanineKall** [liedekef/events-made-easy#514] [General]
We want to send a PDF ticket template sent for each paid booking. We have to manually add the template to each event. 
We have a lot of event managers working with us, and adding the ticket is often forgotten, which leads to a lot of extra work coming from clients asking for a ticket afterwards. 

Is it possible to attach the PDF ticket to the desired email template by default?

**Comment by liedekef** [comment]
That is already possible: in the EME settings => Email templates => Booking paid email
There you can add attachments.

**Comment by JanineKall** [comment]
I noticed, but it seems not possible to default attach the EME PDF ticket template, does it? I mean this one:
![image](https://github.com/liedekef/events-made-easy/assets/129589814/fbbd7e00-8ff8-4a6f-b9a9-c7b8a2611b91)

**Reply by liedekef** [reply]
That field is indeed not set via a default option. And adding template-selection as attachment to each email template is possible, but requires quite some coding. Since I don't see a demand for this (except from you), this is currently not on my todo list.
